python/kvikio/kvikio/cufile_driver.py

```diff
@@ -10,6 +11,24 @@
 DriverProperties = cufile_driver.DriverProperties
 
 
+def libcufile_version() -> Tuple[int, int]:
+    """Get the libcufile version (or zero if older than v1.8).
```

Comment:
Minor note that this returns (0, 0) and not 0 for old versions. I would suggest removing the parenthetical here and instead specifying that behavior in the Notes below.

Reply:
Changed to:
```python
def libcufile_version() -> Tuple[int, int]:
    """Get the libcufile version.

    Returns (0, 0) for cuFile versions prior to v1.8.

    Notes
    -----
    This is not the version of the CUDA toolkit. cufile is part of the
    toolkit but follows its own version scheme.

    Returns
    -------
    The version as a tuple (MAJOR, MINOR).
    """
```